src/lib/styles/material/_table.scss

```diff
@@ -9,6 +9,20 @@
 {
     box-shadow: var(--box-shadow);
     
+    @media (max-width: 992px)
+    {
+        overflow: auto;
+    }
+    
+    .mat-row,
+    .mat-header-row
+    {
+        @media (max-width: 992px)
```

Comment:
Wieso läuft denn das per media query? Ist das nicht eigentlich abhängig von der Breite der Tabelle?

Reply:
wie willste denn in css herausfinden wie breit die Tabelle ist?

Reply:
Das muss man doch garnicht. `overflow-x: auto` und ne min-width erledigen doch das ganze, oder nicht?

Reply:
Was sagen denn @vwiebe oder @zakpur dazu?